docs/development-process/standard-release-process.md

```diff
@@ -27,7 +27,7 @@ process will be described in layer-specific documentation.
 4. Commit and push the changelog.
 5. Merge the changes from `develop` into `main`.
 6. Create and push a tag to `main` that matches the newly bumped version, with
-   `v` added at the beginning (e.g. `v2.0.1-rc1`).
+   `v` added at the beginning (e.g. `v2.0.1-rc.1`).
```

Comment:
@PLangowski @m-iwanicki what about artifacts/version in the release tab? Should we fix the CI scripts as well? https://github.com/Dasharo/meta-dts/releases/tag/v2.0.1-rc3

Reply:
Those version are created from tags so no.